### PR TITLE
Feature/attendance form

### PR DESCRIPTION
**Pull request by ryan-hamblin**
Attendance is going to be an iterative feature. For now we have a stub for the api calls, date picker, selectEvent, and CSV uploader. The uploader has by far been the hardest part of all of this. 

This also refactors our types for our functions. I think we finally have it down so that we can get away from `any`. 

The feature includes a lot of boilerplate code which is why we're logging a few things in lieu of finishing the data flow. I think we're at a good place with the UI that will help us jump-start the development of the apps.

### CODE REVIEW

**Comment by ryan-hamblin** [comment]
image

File: .eslintrc.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = {
     ],
     "no-shadow": "off",
     "@typescript-eslint/no-shadow": ["error"],
-    "max-len": ["warn", { code: 80 }],
+    "max-len": ["warn", { code: 800 }],
```
Comment (ryan-hamblin) [inline .eslintrc.js:34]: Please don't judge me.

Comment (paulstgermain) [reply .eslintrc.js:34]: No judgment here - this 80 character limit + TypeScript was killing me too!

File: src/App.tsx [hunk 2]
```diff
@@ -12,26 +13,32 @@ import LearnerSuccess from "./components/LearnerSuccess/LearnerSuccess";
 import FourthComponent from "./components/FourthComponent/FourthComponent";
 import Home from "./components/Home/Home";
 
-const App = (): any => (
+const { Content } = Layout;
+
+const App = (): JSX.Element => (
```
Comment (paulstgermain) [inline src/App.tsx:18]: Nice to see these `any`s translated into the proper return types for our React components!

**Review by paulstgermain — APPROVED** [review]
This is a solid PR that wireframes up the form and starts the functionality that we're looking for for our Attendance Tracker functionality - code is clean and consistently organized as always, and with the solid notes left here plus said wireframing, we have a fantastic jumping-off point for arranging the specific values to be sent and received from backend when the time comes.

I'd love to see a screenshot of the problem the compiler is giving you with the date picker, and moreso to consider alternatives if this just isn't working. Also - and this isn't merge-blocking at all, just a note for my own reference to discuss later - are we still passing the Zoom API a date with Ash's current course with the backend?

**Pull request merged**